tests/qam-updinstall/update_install.pm

```diff
@@ -390,6 +390,12 @@ sub run {
             }
         }
 
+        if (is_s390x) {
+            # Make sure that openssh-server-config-disallow-rootlogin is not installed
+            # since in s390 we need to ssh to the system to reconnect to the tty after a reboot
+            zypper_call("rm openssh-server-config-disallow-rootlogin", exitcode => [0, 104], log => "zypper_$patch.log", timeout => 1500);
```

Comment:
Please remove the `log` part, not related to path log.

Reply:
done